app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -285,6 +263,8 @@ private function getOrCreateUser($record, $values)
 
             if ($user->id) {
                 info('created user', ['user' => $user->id]);
+            } else {
+                throw new HttpException(500, 'Unable to create user: ' . $record['Email address']);
```

Comment:
Instead of an `HttpException`, let's just throw a normal `\Exception`. HttpExceptions are automatically caught by Laravel to render as a nice error page, but since this isn't happening in a web process that doesn't do much for us.